packages/valory/skills/proposal_collector_abci/rounds.py

```diff
@@ -74,7 +74,12 @@ def proposals(self) -> dict:
     @property
     def votable_proposal_ids(self) -> set:
         """Get the votable proposal ids, sorted by their remaining blocks until expiration, in ascending order."""
-        return set(self.db.get("votable_proposal_ids", {}))
+        return cast(set, self.db.get("votable_proposal_ids", {}))
```

Comment:
Why that change?

Reply:
Linting issues. Would you like me to revert it?